app/bundles/EmailBundle/Swiftmailer/Momentum/Service/SwiftMessageService.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace Mautic\EmailBundle\Swiftmailer\Momentum\Service;
+
+use Mautic\EmailBundle\Helper\PlainTextMassageHelper;
+use Mautic\EmailBundle\Swiftmailer\Message\MauticMessage;
+use Mautic\EmailBundle\Swiftmailer\Momentum\DTO\TransmissionDTO;
+use Mautic\EmailBundle\Swiftmailer\Momentum\Metadata\MetadataProcessor;
+use Symfony\Component\Translation\TranslatorInterface;
+
+/**
+ * Class SwiftMessageService.
+ */
+final class SwiftMessageService implements SwiftMessageServiceInterface
+{
+    /**
+     * @var TranslatorInterface
+     */
+    private $translator;
+
+    /**
+     * MomentumSwiftMessageService constructor.
+     *
+     * @param TranslatorInterface $translator
+     */
+    public function __construct(
+        TranslatorInterface $translator
+    ) {
+        $this->translator = $translator;
```

Comment:
Translator is not used in the whole class.

Reply:
Replaced by Logger (actually used this time :) )